styleguide.config.js

```diff
@@ -1,4 +1,10 @@
 const path = require('path');
+
+const webpackConfig = require('./webpack.config');
+webpackConfig.resolve = webpackConfig.resolve || {};
+webpackConfig.resolve.alias = webpackConfig.resolve.alias || {};
+webpackConfig.resolve.alias['ic-snacks'] = path.join(__dirname, 'dist', 'snacks.js');
```

Comment:
resolving `'ic-snacks'` to built package. Makes readmes more readable & sensical